### ISSUE TITLE
0.12.0 installation issue -- composer error

### ISSUE THREAD

**Issue by jSML4ThWwBID69YC** [pixelfed/pixelfed#5062]
Hello, 

I am trying to install 0.12.0 from scratch. I've run into a composer error that is not making much sense. Here's my setup, and the error. 

**Basic setup steps.**

```
git clone https://github.com/pixelfed/pixelfed.git pixelfed
cd pixelfed/
git checkout v0.12.0
composer install --no-ansi --no-interaction --optimize-autoloader
```

**Error**
Composer runs and appears to work fine until the end. During the optimize-autoload step, the following error occurs. 

```
Generating optimized autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover --ansi

   Illuminate\Database\QueryException 

  SQLSTATE[HY000] [2002] Permission denied (Connection: mysql, SQL: select * from `config_cache` where `k` = pixelfed.oauth_enabled limit 1)

  at vendor/laravel/framework/src/Illuminate/Database/Connection.php:829
    825▕                     $this->getName(), $query, $this->prepareBindings($bindings), $e
    826▕                 );
    827▕             }
    828▕ 
  ➜ 829▕             throw new QueryException(
    830▕                 $this->getName(), $query, $this->prepareBindings($bindings), $e
    831▕             );
    832▕         }
    833▕     }

  1   [internal]:0
      Illuminate\Foundation\Application::Illuminate\Foundation\{closure}(Object(App\Providers\AuthServiceProvider))

      +20 vendor frames 
  22  app/Services/ConfigCacheService.php:153
      Illuminate\Database\Eloquent\Builder::first()
Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 1
```

That makes little sense since we've not configured the database yet. Do we need to configure the .env file before running composer now? 

**Additional details**
OS: FreeBSD 13.3
PHP: 8.2
Redis: 7.2.4

Special note: Upgraded from 0.11.2 to 0.12.0 works as expected. Composer only throws an error on a clean installation.

Any advice or ideas welcome. I'm happy test and try things.

**Comment by jSML4ThWwBID69YC** [comment]
Hello, 

There is a simple workaround. 

**Broken**
```
git clone https://github.com/pixelfed/pixelfed.git pixelfed
cd pixelfed/
git checkout v0.12.0
composer install --no-ansi --no-interaction --optimize-autoloader
```

**Works**
```
git clone https://github.com/pixelfed/pixelfed.git pixelfed
cd pixelfed/
git checkout v0.12.0
cp .env.example .env
composer install --no-ansi --no-interaction --optimize-autoloader
```

My guess is the _ENABLE_CONFIG_CACHE=false_ line must be present in the .env to avoid the issue. 

Reference: 
https://github.com/pixelfed/pixelfed/issues/5046
https://github.com/pixelfed/pixelfed/commit/6263e90a136728a55903a8113ebb9ccbe365a9b6

**Comment by ghost** [comment]
Even though the issue I raised has been closed, it has not been improved in the next version, which makes me very disappointed.

**Comment by dansup** [comment]
@7shironana I'm in the process of releasing the fix and bumping the version, will update this when thats done!

**Comment by dansup** [comment]
The fix is shipped in the latest version (v0.12.1). Feel free to re-open this if you are still experiencing these issues!
